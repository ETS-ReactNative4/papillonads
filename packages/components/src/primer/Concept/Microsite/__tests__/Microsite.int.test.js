import {
  renderSpecificationsStandardsGuidelines,
  renderOpenSourceDependencyProjects,
  renderDesignSystemCore,
  renderProductMicrosites,
} from './Microsite.int.render'

describe('<Microsite />', () => {
  describe('Render', () => {
    test('must match renderSpecificationsStandardsGuidelines()', () => {
      expect(global.renderToJSON(renderSpecificationsStandardsGuidelines())).toMatchSnapshot()
    })
    test('must match renderOpenSourceDependencyProjects()', () => {
      expect(global.renderToJSON(renderOpenSourceDependencyProjects())).toMatchSnapshot()
    })
    test('must match renderDesignSystemCore()', () => {
      expect(global.renderToJSON(renderDesignSystemCore())).toMatchSnapshot()
    })
    test('must match renderProductMicrosites()', () => {
      expect(global.renderToJSON(renderProductMicrosites())).toMatchSnapshot()
    })
  })
})
