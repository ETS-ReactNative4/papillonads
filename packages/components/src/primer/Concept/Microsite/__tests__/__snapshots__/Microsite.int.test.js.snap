// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Microsite /> Render must match renderDesignSystemCore() 1`] = `
<div
  className={null}
>
  <h2>
    ðŸ“— Papillon Ads Implementation
  </h2>
  <h3>
    ðŸ“–. Design System Core
  </h3>
  <h4>
    ðŸ“ƒ. Setup file(s)
  </h4>
  <div>
    ï¹˜ Coming soon.
  </div>
  <h4>
    ðŸ“ƒ. Knowledge base
  </h4>
  <div>
    ï¹˜ Coming soon.
  </div>
</div>
`;

exports[`<Microsite /> Render must match renderOpenSourceDependencyProjects() 1`] = `
<div
  className={null}
>
  <h2>
    ðŸ“— Open Source Dependency Projects
  </h2>
  Coming soon...
</div>
`;

exports[`<Microsite /> Render must match renderProductMicrosites() 1`] = `
<div
  className={null}
>
  <h2>
    ðŸ“— Papillon Ads Implementation
  </h2>
  <h3>
    ðŸ“–. Product Microsites
  </h3>
  <h4>
    ðŸ“ƒ. Setup file(s)
  </h4>
  <div>
    ï¹˜ Coming soon.
  </div>
  <h4>
    ðŸ“ƒ. Knowledge base
  </h4>
  <div>
    ï¹˜ Coming soon.
  </div>
</div>
`;

exports[`<Microsite /> Render must match renderSpecificationsStandardsGuidelines() 1`] = `
<div
  className={null}
>
  <h2>
    ðŸ“— Specifications/Standards/Guidelines
  </h2>
  Coming soon...
</div>
`;
